### PR TITLE
ready for cloudflare pages

### PR DESCRIPTION
**Pull request by hobbitronics**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "prebuild:dev": "npm run clean:build",
     "build:dev": "rollup -c -w",
-    "prebuild:prod": "npm run clean:build",
+    "prebuild:prod": "echo ANALYTICS_ID=${ANALYTICS_ID} >> .env; echo ANSWER=${ANSWER} >> .env; echo API_HOST=${API_HOST} >> .env; echo THE_ONE=${THE_ONE} >> .env; npm run clean:build",
```
Comment (briskt) [inline package.json:7]: Is this applicable here in this template?

Comment (hobbitronics) [reply package.json:7]: yes, those are the variables for the .env example. You can see they are working [here](https://65f430fc.the-way-svelte-cwa.pages.dev/)

Comment (briskt) [reply package.json:7]: Cool. I didn't realize this template actually deployed.

**Review by briskt — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (hobbitronics) [reply package.json:7]: now I just need to be able to switch env variables based on branch.

**Pull request merged**